packages/ibm-products-web-components/vite.config.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { defineConfig } from 'vitest/config';
+import { litStyleLoader, litTemplateLoader } from '@mordech/vite-lit-loader';
+import sass from 'sass';
```

Comment:
```suggestion
import * as sass from 'sass';
```

Sass will complain that importing this way is deprecated.